plot_app/templates/main.js

```diff
@@ -7,6 +7,30 @@ $.fn.scrollView = function () {
     });
 }
 
+$(".chosen-select").chosen({
+    no_results_text: "Oops, nothing found!",
+    width: "100%"
+  });
+
+ $( "#error-label" )
+  .change(function () {
+    var error_labels = "";
+    $( "select option:selected" ).each(function() {
+        error_labels += $( this ).text() + ",";
+       //error_labels.push($( this ).text())
+    });
+
+
+    $.ajax({
+        type: "POST",
+        url: "http://localhost:5006/error_label",
```

Comment:
This points to localhost. You can use something like `window.location.protocol+'//'+window.location.host+'/error_label`

Reply:
thank you for pointing that out, I forgot to change that. Simply using the entry point "/error_label" works here or do you prefer the solution with window.location?

Reply:
No, the simpler, the better.